docs/_scripts/execute_notebooks.sh

```diff
@@ -23,10 +23,10 @@ execute_notebook() {
 export -f execute_notebook
 
 # Find all notebooks and filter out those in the skip list
-notebooks=$(find docs/docs/tutorials docs/docs/how-tos -name "*.ipynb" | grep -v ".ipynb_checkpoints" | grep -vFf <(echo "$SKIP_NOTEBOOKS"))
+# notebooks=$(find docs/docs/tutorials docs/docs/how-tos -name "*.ipynb" | grep -v ".ipynb_checkpoints" | grep -vFf <(echo "$SKIP_NOTEBOOKS"))
+notebooks=$(find docs/docs/tutorials -name "*.ipynb" | grep -v ".ipynb_checkpoints" | grep -vFf <(echo "$SKIP_NOTEBOOKS"))
 
-# Run notebooks in parallel
-if ! parallel execute_notebook ::: $notebooks; then
```

Comment:
Out of curiosity why remove `parallel`?

Reply:
mainly because it has unexpected side effects when running with VCR locally

Reply:
(and it actually has no effect on the runner since it doesn't have multiple core, although maybe it's possible to enable that)